produkt-bundles/4445-10040-Sorglos-ohne-TSE.md

```diff
@@ -6,7 +6,7 @@ Dieses Bundle inkludiert folgende Produkte:
 
 -  [4445-0003-lokal-installierte-middleware](../compliance-as-a-service/produkte/4445-0003-lokal-installierte-middleware.md) 
 -  [4445-100XX-pos-archiv](../revisionssichere-daten-as-a-service/produkte/4445-100XX-pos-archiv.md) 
--  [4445-10020-Audicon-Kassenarchiv-Online](../revisionssichere-daten-as-a-service/produkte/4445-10020-Audicon-Kassenarchiv-Online-md) 
+-  [4445-10020-Audicon-Kassenarchiv-Online](../revisionssichere-daten-as-a-service/produkte/4445-10020-Audicon-Kassenarchiv-Online.md) 
```

Comment:
Should the links be absolute, not relative so they can be used on other locations (e.g. docs.fiskaltrust.cloud)?